### ISSUE TITLE
biblatex error

### ISSUE THREAD

**Issue by muldead** [thomasjo/atom-latex#443]
Hello, I'm having difficulties printing out a bibliography with the biblatex package. I'm getting an 'empty bibliography' message when I compile a simple .tex document in atom (attached below). I've also attached the .bib file and the .log file from running. I'm using tex live 2017 and compiling with pdflatex. I am a latex and atom novice so I would appreciate even the most basic levels of advice. 

Regards
Andrew 

[sample_bib_log.log](https://github.com/thomasjo/atom-latex/files/1520193/sample_bib_log.log)
[sample_bib.txt](https://github.com/thomasjo/atom-latex/files/1520196/sample_bib.txt)
[bibbib.txt](https://github.com/thomasjo/atom-latex/files/1520197/bibbib.txt)

**Comment by maecky** [comment]
I get the same error. I'm using the latest MikTex on Windows 10 64 bit.

**Comment by maecky** [comment]
I managed to fix it. Have a look in your *.blg file.
Mine had this line in it:
`[176] Utils.pm:180> ERROR - Error: Found biblatex control file version 3.4, expected version 3.3.`

First I tried to update bibtex and biber but that didn't help. Then I started the MikTex Settings (Admin) and executed the two maintenance commands under the tab "General" -> "Refresh FNDB" and "Update Formats".

Then I cleaned the folder and build the project again. Now it is working.

**Comment by yitzchak** [comment]
@maecky Thanks for helping out! Looks like the latest version of MiKTeX shipped with an updated version of the BibLaTeX package but not an updated version of Biber.

@muldead If you use `addbibresource` make sure you include the file extension. For instance, to refer to `foo.bib` you should use

```latex
\addbibresource{foo.bib}
```

If you use the `bibliography` command you can omit the extension

```latex
\bibliography{foo}{}
```

**Comment by muldead** [comment]
Awesome. I cleaned the folder and tweaked the addbibresource input and it seems to be working out. Thanks so much!  Follow on question, then I'm done. I now have installed both miktex and tex live. Is there any reason to use one or the other to compile documents in atom?

**Comment by yitzchak** [comment]
**TL;DR** SInce @thomasjo and myself both use TeX Live/MacTeX primarily, `atom-latex` usually gets more testing time on TeX Live, hence I suspect both of us would recommend TeX Live.

As far as MiKTeX versus TeX Live in general, I can only speak to my own experience and please keep in mind I am on Linux and only use Windows occasionally. Those that use Windows as their primary platform may have a different perspective than myself since MiKTeX is primarily designed for Windows.

The feature that MiKTeX has over TeX Live most frequently referred to as a distinct advantage is the automatic installation of packages. Personally, I find this feature to be unneeded given the size of storage devices now days. I believe it is easier to do a full TeX Live install and just run the update facility occasionally as it will pull directly from CTAN.

To be completely honest, I avoid MiKTeX if possible. For myself it is not really about features, but about reliability and the ability get issues such as bugs resolved. On both fronts I have had a plethora of negative experiences with MiKTeX.

**Comment by muldead** [comment]
sold. thanks again
